core/src/main/scala/chisel3/experimental/Trace.scala

```diff
@@ -46,7 +52,29 @@ object Trace {
     }
   }
 
-  /** An Annotation that records the original target annotate from Chisel.
+  /** Trace an Instance name. */
+  def traceNameV2(x: RawModule): Unit = {
+    annotate(new ChiselAnnotation {
+      def toFirrtl: Annotation = TraceAnnotation(x.toAbsoluteTarget, x.toAbsoluteTarget)
+    })
+  }
+
+  /** Trace a Data name. This does NOT add "don't touch" semantics to the traced data. If you want this behavior, use an explicit [[chisel3.dontTouch]]. */
+  def traceNameV2(x: Data): Unit = {
+    x match {
+      case aggregate: Aggregate =>
+        annotate(new ChiselAnnotation {
+          def toFirrtl: Annotation = TraceAnnotation(aggregate.toAbsoluteTarget, aggregate.toAbsoluteTarget)
+        })
+        aggregate.getElements.foreach(traceNameV2)
```

Comment:
`traceName` does not do any element splitting for aggregates, why does `traceNameV2`?

Reply:
This is hiding in part of the collapsed diff. No change was made. Original version is here: https://github.com/chipsalliance/chisel3/pull/2786/files#diff-37ec504e20de8b25bdfa4e671e88ee14a1f83fdf941eedb658e9ed6664d5c185R47